items/armor/armor.json

```diff
@@ -439,17 +409,17 @@
 		"stamina_regen": 100,
 		"passive": 0
 	},
-	"4107448458": {
```

Comment:
This ID seems changed / no longer recurring?

Reply:
Was the items mixID, changed to item ID of "2891794937"